soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -1340,6 +1340,33 @@ struct peci_it8xxx2_regs {
 #define CECOPSTS		ECREG(EC_REG_BASE_ADDR + 0x2E06)
 #define CECCRH			ECREG(EC_REG_BASE_ADDR + 0x2E07)
 
+/**
+ *
+ * (37xxh, 38xxh) USBPD Controller
+ *
+ */
+/* TODO: After pd driver is implemented, access these registers by reg property. */
+#define IT8XXX2_USBPD_BASE(port)	(0x00F03700 + (0x100 * (port) * (port)))
```

Comment:
This certainly looks like it should be moved to the device tree. I understand (looking at the rest of this file) that this is how it was done before, but maybe we shouldn't continue this practice?

Reply:
We will re-visit here and move it to .dtsi file.

Reply:
Ping. This hasn't been addressed yet.

Reply:
Done